XamlControlsGallery/XamlControlsGallery.DesktopWap.csproj

```diff
@@ -75,6 +75,9 @@
     <Compile Remove="AnimatedVisuals\LottieLogo1.cs" />
     <Compile Remove="ControlPages\AnimatedVisualPlayerPage.xaml.cs" />
 
+    <!-- Disabled until Reunion builds Microsoft.WinUI.dll against CSWinRT 1.1.4 -->
+    <Compile Remove="ControlPages\XamlCompInteropPage.xaml.cs" />
```

Comment:
Why is this Compile remove instead of Page remove?

Reply:
This is removing the code-behind for the page, the page itself is removed further down on line 92

Reply:
D'oh, I see now. Thanks.